plugin/pulpcore/plugin/download/http.py

```diff
@@ -54,6 +54,10 @@ class HttpDownloader(BaseDownloader):
     allow for an active download to be arbitrarily long, while still detecting dead or closed
     sessions even when TCPKeepAlive is disabled.
 
+    If a session is not provided, the one created will disable HTTP session force TCP closure with
```

Comment:
s/will disable HTTP session force TCP closure with each request/will force TCP connection closure after each request/

Reply:
fixed in next push, ty